### ISSUE TITLE
Addon sugguests groups that you've already killed.

### ISSUE THREAD

**Issue by Truth-** [Hiale/HellbaneHelper#1]
The list of remaining keywords isn't updated properly. It's possible for the addon to autojoin a group for a mob you just killed.
My suggestion would be to force update the keyword list in GetRemainingUnits function disregarding the updateKeywords variable.
![broken](https://cloud.githubusercontent.com/assets/5549305/8728745/3bc8cb9a-2be7-11e5-89c7-1430d6715737.png)

**Comment by Hiale** [comment]
Yes, I know that. But the problem is even different. While you are fighting a certain mob, a new group could be found and the addon tries to join, which probably will fail because you are not the leader. I decided to leave this issue for the moment. Because when you change subzones, the list gets updated anyway.
